docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 db:
-  image: postgres:9.4.1
+  image: postgres:9.4.4
   ports:
-    - "5432:5432"
+    - "5433:5432"
```

Comment:
I didn't even realize the ports in the container could clash with the host ports until they did. Not quite like a VM :)